device/core/readme.md

```diff
@@ -3,23 +3,34 @@ The core components of the Azure IoT device SDK.
 
 [![npm version](https://badge.fury.io/js/azure-iot-device.svg)](https://badge.fury.io/js/azure-iot-device)
 
+## Features
+
+Use the Azure IoT device SDK to:
+* Send event data to Azure IoT Hub.
+* Receive messages from IoT Hub.
+* Communicate with the service via MQTT (optionally over WebSockets, AMQP (optionally over WebSockets),  or HTTP.
+* Synchronize an Azure IoT Hub device Twin with Azure IoT Hub from a device
+* Implement Azure IoT Hub Direct Device Methods on devices
+* Implement Azure IoT Device Mangement features on devices
+
 ## Prerequisites
 You need to install **Node.js** JavaScript runtime environment for Azure IoT JavaScript client SDK to run on your platform. For downloading Node.js refer to [Node.js][nodejs_lnk]. To check if your platform (OS) is **supported** check if install package is available at [Node.js][nodejs_dwld_lnk].
 [npm][npm_lnk] is a package manager command line tool that will be included when Node.js is installed. This is the tool that you will use to install Azure IoT node.js client side SDK.
 
-
 ## Install
 
 `npm install -g azure-iot-device@latest` to get the latest (pre-release) version.
```

Comment:
nope, @latest will get the latest stable. and since device management is released there are on pre-release packages anymore.